### ISSUE TITLE
Does ews-api support aliases?

### ISSUE THREAD

**Issue by mhodovaniuk** [OfficeDev/ews-java-api#684]
I am using `DOMAIN\UserID\Alias` as username for `findItems` operation and getting `(401)Unauthorized`. At the same time, `DOMAIN\UserID\Alias` works fine with IMAP. It seems that ews-api does not support aliases. Can someone please approve that it is not possible to use alias as username? Or maybe I am using the wrong format?

Thanks in advance.

**Comment by pkropachev** [comment]
Do you use such format for `WebCredentials`?
If to be honest, I've never heard about such way of logon (either samAccountName or userPrincipalName (UPN)).

**Comment by mhodovaniuk** [comment]
Yes, I am trying to authenticate with `WebCredentials`. @pkropachev so looks like it is not possible. Thank you for the response.
